test/e2e/testing-manifests/auth/encrypt/connect-registry.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# input and default values
+KIND_NETWORK="${1:-kind}"
+REGISTRY_NAME="${2:-kind-registry}"
+
+if [ "${KIND_NETWORK}" != "bridge" ]; then
```

Comment:
We can get rid of this `if` check as `bridge` network has been dropped by kind a while back.